modules/brew/README.md

```diff
@@ -0,0 +1,136 @@
+# brew
```

Comment:
I just remembered that `brew.sh` is included in `ublue-os-just` RPM package.
It resides in `/usr/etc/profile.d/brew.sh`

It adds brew in path only when shell is interactive, solving issues with `brew` path clashing with other programs

```bash
#!/usr/bin/env bash
[[ -d /home/linuxbrew/.linuxbrew && $- == *i* ]] && eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
```

This is only in Universal Blue images.

We need to account for other images which don't include this.